Marlin/ultralcd.cpp

```diff
@@ -1848,6 +1849,7 @@ char *ftostr52(const float &x) {
       _lcd_level_bed_position = 0;
       lcd_goto_menu(_lcd_level_bed);
     }
+    lcdDrawUpdate = 2;
```

Comment:
Can't explain why this worked best! Without it the menu function didn't recognize when homing was done.